src/current/v22.2/changefeed-messages.md

```diff
@@ -126,6 +126,8 @@ In some unusual situations you may receive a delete message for a row without fi
 
 ## Schema Changes
 
+In v22.1, CockroachDB introduced the [declarative schema changer](online-schema-changes.html#declarative-schema-changer). When schema changes happen that use the declarative schema changer by default, changefeeds will not emit duplicate records for the table that is being altered. It will only emit a copy of the table using the new schema. Refer to [Schema changes with column backfill](#schema-changes-with-column-backfill) for examples of this.
```

Comment:
```suggestion
In v22.1, CockroachDB introduced the [declarative schema changer](online-schema-changes.html#declarative-schema-changer). When schema changes happen that use the declarative schema changer by default, changefeeds will **not** emit duplicate records for the table that is being altered. It will only emit a copy of the table using the new schema. Refer to [Schema changes with column backfill](#schema-changes-with-column-backfill) for examples of this.
```

adding for clarity and to callout to the user since this is phrased to show behaviour with the declarative schema changer by default while the section below talks about schema changes that don't use the declarative schema changer